src/components/AssetMatching/components/Toolbox/Toolbox.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from 'react';
+import { MatchingAction } from '../../types';
+import PrimaryActions from './PrimaryActions';
+import SecondaryActions from './SecondaryActions';
+import { flex } from '../../../../theme/functions';
+
+interface Props {
+  matchingActions?: MatchingAction[];
+}
+
+const Toolbox: FC<Props> = ({ matchingActions = [] }) => {
+  return (
+    <div css={flex}>
+      <PrimaryActions matchingActions={matchingActions} />
```

Comment:
You could be slicing the matchingActions array here. It's nothing major but now to figure out what is a primary and what is a secondary action I need to check inside the Primary and Secondary actions components. Otherwise they could be two different props.